### ISSUE TITLE
(non-Custom) Descriptions of the effects spells in Spell/Character windows don't take Spell Power into account

### ISSUE THREAD

**Issue by firecat6666** [Ruin0x11/ElonaPlusCustom_GX#87]
As was discussed here: https://github.com/Ruin0x11/ElonaPlusCustom-GX/issues/48

Especially now that Magic grants Spell Power and considering the fact that Spell Power affects certain things but not others (and Divine Wisdom seems to only be affected by Spell Power from equipment and not from Magic), it would be great if the descriptions showed the exact numbers for the effects of spells.

However, I'm opening this issue for consideration, as this does seem like something that Ano should fix in the base game instead of it being fixed in Custom. Feel free to close this if you think this shouldn't be fixed in Custom.

**Comment by EnragedWaters** [comment]
Yeah there was originally an exploit when Divine Wisdom accounted for Magic attribute since you could infinite spam divine wisdom for 2000 Magic/Learning so they had to remove it.
However, I have noticed damage spells do in fact account for spell power - it's just buffs like Hero/Concentration/Etc that don't account for spell power and aren't displayed correctly - but their actual math is still fine(100 turn description, 200% spell power, but gives correct 200 turn buff)
Correct me if I'm wrong however, I might have read your post wrong.

**Comment by firecat6666** [comment]
Hmm, I only actually checked the first page of spells. There are some buffs and healing spells there (and some other stuff), I didn't see any difference in the description of any of those spells when changing my spell power.

Just to be sure, I'm talking about the spells descriptions, not the actual effects of the spells, nor the description in the character page for active hexes.
